openlibrary/macros/databarWork.html

```diff
@@ -123,7 +123,7 @@ <h3 class="closed">
     </div>
     </div>
 
-$if isbn or oclc_numbers or 'lendinglibrary' in collection or library:
+$if (isbn or oclc_numbers or 'lendinglibrary' in collection or library) and not any_private_collections(collection):
```

Comment:
I think this may be a naming issue. `collection` is either a `set` or a `list`, never a `string`. This was a variable that already existed. Maybe I should change the name to `collections`.